linera-service/src/cli_wrappers/kubectl.rs

```diff
@@ -6,23 +6,44 @@ use anyhow::{Context, Result};
 use tokio::process::{Child, Command};
 
 pub struct KubectlInstance {
-    pub port_forward_child: Option<Child>,
+    pub port_forward_children: Option<Vec<Child>>,
 }
 
 impl KubectlInstance {
-    pub fn new(port_forward_child: Option<Child>) -> Self {
-        Self { port_forward_child }
+    pub fn new(port_forward_children: Option<Vec<Child>>) -> Self {
+        Self {
+            port_forward_children,
+        }
+    }
+
+    // If there are multiple clusters, make sure you always properly set the context
+    // before doing kubectl commands, otherwise you might be querying the wrong cluster
+    pub async fn config_use_context(&mut self, cluster_id: u32) -> Result<()> {
+        Command::new("kubectl")
+            .arg("config")
+            .arg("use-context")
+            .arg(&format!("kind-{}", cluster_id))
+            .spawn_and_wait_for_stdout()
+            .await?;
+        Ok(())
     }
 
     pub async fn port_forward(&mut self, pod_name: &str, ports: &str) -> Result<()> {
-        self.port_forward_child = Some(
-            Command::new("kubectl")
-                .arg("port-forward")
-                .arg(pod_name)
-                .arg(ports)
-                .spawn()
-                .context("Port forwarding failed")?,
-        );
+        let port_forward_child = Command::new("kubectl")
```

Comment:
How does this command know which context to use?

Reply:
See commend above.

Reply:
Commented above